e2e/fake-timers/clear-real-timers/__tests__/clearRealTimers.test.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+test('allows clearing not faked timers', () => {
+  const timer = setTimeout(() => console.log('should not be called'), 1000);
```

Comment:
not sure what this test does. we don't wait for 1000ms for it to trigger

Reply:
This test makes sure that `shouldClearNativeTimers: true` was passed. The trick is that `setTimeout()` is called before `jest.useFakeTimers()` and it gets cleared. Hence no need to wait for 1000ms. Hm.. would be better to throw an error than `console.log()`. Will fix that and will add a comment.